Dockerfile

```diff
@@ -279,23 +279,26 @@ RUN curl -o- -L https://yarnpkg.com/install.sh > /usr/local/bin/yarn-installer.s
 
 ENV NVM_VERSION=0.39.1
 
-# Install node.js, yarn, grunt, bower and elm
+# Install node.js, yarn, grunt, bower
 USER buildbot
 RUN git clone https://github.com/creationix/nvm.git ~/.nvm && \
     cd ~/.nvm && \
     git checkout v$NVM_VERSION && \
     cd /
 
-ENV ELM_VERSION=0.19.1-5
 ENV YARN_VERSION=1.22.19
+ENV PNPM_VERSION=7.13.4
 
 ENV NETLIFY_NODE_VERSION="16"
 
 RUN /bin/bash -c ". ~/.nvm/nvm.sh && \
          nvm install --no-progress $NETLIFY_NODE_VERSION && \
          npm install -g grunt-cli bower && \
-             bash /usr/local/bin/yarn-installer.sh --version $YARN_VERSION && \
```

Comment:
I guess we could remove the step that downloads this script (line 278), because I guess it would conflict with corepack anyway? OR if we want to keep it, make our own version that uses corepack?

Reply:
Good point thx forgot that